contributors/devel/sig-testing/verify-tests.md

```diff
@@ -0,0 +1,78 @@
+# Verification Tests
+
+**Table of Contents**
+
+- [Verification Tests](#verification-tests)
+  - [Overview](#overview)
+  - [`verify-govet-levee`](#verify-govet-leve)
+
+## Overview
```

Comment:
Can we add an explicit note that this is an incomplete listing currently?

Reply:
Added.

Reply:
I opened https://github.com/kubernetes/community/issues/5532 to track adding docs for the rest